app/models/further_information_request.rb

```diff
@@ -67,8 +67,11 @@ def after_received(*)
   end
 
   def after_expired(user:)
-    if application_form.withdrawn_at.nil?
-      DeclineQTS.call(application_form:, user:)
+    return if application_form.withdrawn_at.present?
+
+    ActiveRecord::Base.transaction do
+      application_form.update!(declined_at: Time.zone.now)
+      ApplicationFormStatusUpdater.call(application_form:, user:)
```

Comment:
Why don't we use the `DeclineQTS` service here anymore?

Without it we won't get this timeline event or decline email.